src/modules/registrypreview/RegistryPreviewUI/RegistryPreviewXAML/App.xaml

```diff
@@ -7,12 +7,9 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
-                <!-- Other merged dictionaries here -->
+                <!--  Other merged dictionaries here  -->
             </ResourceDictionary.MergedDictionaries>
-            <!-- Need to set these to transparent to allow the custom title bar to be its real color -->
-            <SolidColorBrush x:Key="WindowCaptionBackground">Transparent</SolidColorBrush>
```

Comment:
Not a blocker, can be a follow-up PR:
The same thing should be removed from Hosts `App.xaml`. Also, some additional things can be removed, like https://github.com/microsoft/PowerToys/blob/main/src/modules/Hosts/Hosts/HostsXAML/MainWindow.xaml.cs#L31-L38 - the same code is used for few different modules. @niels9001 correct me if I'm wrong here and also point to other workarounds we have that are now solved by 1.4 :)

Reply:
Going to open an issue for that. "Reviewing if workarounds are still needed after upgrading to WASDK 1.4"